Arcade/MinimalistTD/source/Bullet.hx

```diff
@@ -56,10 +56,15 @@ class Bullet extends FlxSprite
 
 		// Move toward the target that was assigned in init().
 
-		if (_target.alive)
+		if (target != null && _target.alive)
```

Comment:
Yes, good catch.

I had made the fix on a different workspace (just MinimalistTD rather than all of demos) but seemed to miss the _ when copying over.